kuma/wiki/jinja2/wiki/list/documents.html

```diff
@@ -12,6 +12,7 @@
   {% set title = _('All documents') %}
 {% endif %}
 {% block title %}{{ page_title(title) }}{% endblock %}
+{% block robots_value %}noindex, nofollow{% endblock %}
```

Comment:
Opened [bug 1364963](https://bugzilla.mozilla.org/show_bug.cgi?id=1364963). I focused on just the tag indexes. I think other indexes, such as pages with KS errors and that need review, can continue being excluded from search indexes.